Vagrantfile

```diff
@@ -11,8 +11,8 @@ end
 # to be able to modify tests outside of vagrant, we use nfs mount point, for more
 # information refer https://www.vagrantup.com/docs/synced-folders/nfs
 Vagrant.configure("2") do |config|
-    config.vm.box = "generic/ubuntu2004"
-    config.vm.synced_folder ".", "/vagrant", type: "nfs"
+    config.vm.box = "generic/ubuntu2204"
+    config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_udp: false
```

Comment:
**Note:** UDP has been disabled for NFS mounts since Ubuntu `20.10` (https://discourse.ubuntu.com/t/jammy-jellyfish-release-notes/24668#udp-disabled-for-nfs-mounts-7)